lsp-mode.el

```diff
@@ -60,6 +60,20 @@
 (require 'yasnippet nil t)
 (require 'lsp-protocol)
 
+;; Not defined prior to Emacs 26.1
+(eval-and-compile
+  (unless (fboundp 'seq-map-indexed)
+    (defun seq-map-indexed (fn sequence)
+      "Return the result of applying FN to each element of SEQUENCE.
+Unlike `seq-map', FN takes two arguments: the element of
+the sequence, and its index within the sequence."
+      (let ((index 0))
+        (seq-map (lambda (elt)
+                   (prog1
+                       (funcall fn elt index)
+                     (setq index (1+ index))))
+                 sequence)))))
```

Comment:
Can't we just use [-map-indexed](https://github.com/magnars/dash.el#-map-indexed-fn-list)?